onnxscript/function_libs/torch_aten/ops/nn.py

```diff
@@ -239,17 +239,26 @@ def aten_conv_depthwise3d(
     raise NotImplementedError()
 
 
-def aten_cross_entropy_loss(
-    self: TensorType,
-    target: TensorType,
-    weight: Optional[TensorType] = None,
-    reduction: int = 1,
-    ignore_index: INT64 = -100,
+@torch_op("aten::cross_entropy")
```

Comment:
Isn't it `aten::cross_entropy_loss`?

Reply:
yes, originally it is cross_entropy_loss, but I cannot find any torch document, only find cross_entropy().

Reply:
![image](https://user-images.githubusercontent.com/11205048/219820844-a9137c0a-e04c-44e3-b224-0be988276b8b.png)

Reply:
I speculate there's probably no documentation for the aten op if it couldn't be found. I would refer to the corresponding symbolic function implementation in torch.onnx as a hint

Reply:
so we will keep cross_entropy_loss() as the function name?

Reply:
Yeah we should keep the original.

Reply:
OK